### ISSUE TITLE
Cannot attach debugger to app service with VS 2022

### ISSUE THREAD

**Issue by clement911** [MicrosoftDocs/visualstudio-docs#7674]
I am able to deploy my app service from VS 2022 without any issue.
However, when I click 'Attach debugger' I get an error: 'Azure resource is not available'

![image](https://user-images.githubusercontent.com/3426504/154152585-a6123e82-ece3-44d1-896a-94c62df12427.png)

![image](https://user-images.githubusercontent.com/3426504/154152674-712cbfc1-4c01-4a4c-9673-6ec84741d857.png)

**Comment by clement911** [comment]
I finally figured out that I had to reenter my azure credentials under Help > Registration.
Only to get another error saying that the remote debugger was not enabled. After researching more, I found out that remote debugging is not possible on Premium V2 tier. 
I tried again on another app service on S1 tier.
When I click 'Attach debugger', it looks like it's doing something and showing 'Attaching the debugger...' but nothing happens and no message is displayed.
Truly a poor experience so  far.

**Comment by WilliamAntonRohm** [comment]
@clement911 -- Clement, thank you for your feedback.  Are you following a particular docs.microsoft.com article? 
 Meanwhile, please consider these resources:
- [Visual Studio on Microsoft Q&A](https://docs.microsoft.com/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

You're welcome to add this here:
- Report product issues to us via the [Report a Problem](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio) tool that appears both in the Visual Studio Installer and in the Visual Studio IDE.​ 
- Suggest a feature, track product issues, and find answers in the [Visual Studio Developer Community](https://developercommunity.visualstudio.com).

**Comment by clement911** [comment]
Yes, I am following https://docs.microsoft.com/en-us/visualstudio/debugger/remote-debugging-azure?view=vs-2022#remote_debug_azure_app_service

**Comment by WilliamAntonRohm** [comment]
@clement911 -- Clement, thank you for your follow-up.

@Mikejo5000 -- Mike, please look into this potential documentation issue.
